source/guide_searx.rst

```diff
@@ -0,0 +1,242 @@
+.. highlight:: console
+
+.. author:: FM <git.fm@mmw9.de>
+
+.. tag:: web
+.. tag:: python
+
+.. sidebar:: Logo
+
+
+  .. image:: _static/images/searx.png
+      :align: center
+
+#####
+searX
+#####
+
+.. tag_list::
+
+Searx_ (Wikipedia_) is a free and open-source metasearch engine which aggregate results from more than 70 search services (e.g. Google, Bing etc.) and to avoid user tracking and profiling by these ones.
+
+There are public instances_ of searx available, to get a personal practice before you install your own.
+
+License
+=======
+
+Searx is released under the `GNU Affero General Public License`_.
+
+----
+
+.. note:: For this guide you should be familiar with the uberspace basic concepts of
+
+  * supervisord_
+  * domains_
+  * `web backends`_
+
+Installation
+============
+
+Step 1 - Repository Cloning
+---------------------------
+
+We will prepare the destination for the searx repository:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ mkdir -p ~/opt/searx/
+
+And clone the repository from GitHub:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ git clone https://github.com/searx/searx.git ~/opt/searx/
+
+Step 2 - Python Module Installation
+-----------------------------------
+
+----
+
+.. note:: Here we use python 3. This is the reason, that these guide consider this version aspect to the installation tool like ``pip`` and later to ``python`` it self.
+
+Some python modules are necessary and will be installed in your uberspace with ``pip3`` and the option ``--user``:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ pip3 install --user requests pyyaml pygments werkzeug babel flask flask_babel lxml langdetect python-dateutil
```

Comment:
This does not ensure the correct versions are installed. Luckily, searx comes with a [`requirements.txt` file](https://github.com/searx/searx/blob/master/requirements.txt) specifying the needed packages as well as their exact versions. Using that one instead of documenting the list here would make the guide more resilient to future updates and easier to use.

What do you think?
```suggestion
 [isabell@stardust ~]$ pip3 install --user ~/opt/searx/requirements.txt
 ```

Reply:
Yes, you have right. But it was necessary to consider additional options too.